devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -14,7 +14,7 @@
  *
  */
 metadata {
-	definition(name: "Aeotec Doorbell Siren 6", namespace: "smartthings", author: "SmartThings", mcdSync: true,  mnmn: "SmartThings", vid: "generic-8-sound") {
+	definition(name: "Aeotec Doorbell Siren 6", namespace: "smartthings", author: "SmartThings", mcdSync: true) {
```

Comment:
One thing I would recommend checking is installing a device with the DTH as it currently is in production (maybe best to self-publish so that you aren't changing the device type through the edit device) and then self-publishing this change to make sure that the new vid is used. I *think* on the DTH level there aren't problems, but I seem to recall issues with deleting a vid from a DTH where even if we were just relying on the DTH name we still had to declare it so that it would update for devices that were already installed.

Reply:
If that doesn't seem to be an issue then we should be fine to merge this.

Reply:
I left a comment under metadata MR. 
Should we use separately metadata for Siren and Doorbell?
I mean for Siren metadata with Alarm capability and for a Doorbel with Chime.

Reply:
@dkirker ping

Reply:
@MGoralczykS I haven't had a chance to look over the implications of either route.

Reply:
@dkirker any updates :)?

Reply:
@MGoralczykS I'm impartial to either way. Since we implement both devices the same way, and they are the same actual device but with a different fingerprint, it might make sense to just use the same metadata. If you want another opinion, you can check with product.

Reply:
@dkirker I think it's fine, then. Can you verify the metadata is out there and merge?